src/core/meshcop/border_agent.hpp

```diff
@@ -253,6 +255,7 @@ class BorderAgent : public InstanceLocator, private NonCopyable
 
     static constexpr uint16_t kUdpPort          = OPENTHREAD_CONFIG_BORDER_AGENT_UDP_PORT;
     static constexpr uint32_t kKeepAliveTimeout = 50 * 1000; // Timeout to reject a commissioner (in msec)
+    otBorderAgentCounters     mCounters;
```

Comment:
Move down next to other member variable definintions?

Reply:
Done